benchmarks/graphics_pipeline/main.cpp

```diff
@@ -45,14 +45,27 @@ class FreeCamera
         BACKWARD
     };
 
-    FreeCamera(float3 eyePosition, float3 target)
+    FreeCamera(float3 eyePosition, float3 target, float theta, float phi)
     {
         mEyePosition = eyePosition;
         mTarget      = target;
+        mTheta       = theta;
```

Comment:
Should this assert that the values are within the range you have in the comment below? Or does it not matter?